ci-operator/step-registry/ipi/conf/vsphere/disconnected/registry/ipi-conf-vsphere-disconnected-registry-commands.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+mirror_registry_url="$(< /var/run/vault/vsphere/vmc_mirror_registry_url)"
+
+#save mirror registry url
+cat > "${SHARED_DIR}"/mirror_registry_url << EOF
+${mirror_registry_url}
+EOF
+
+#Get haproxy-router image for upi disconnected installation
+target_release_image="${mirror_registry_url}/${RELEASE_IMAGE_LATEST#*/}"
+target_release_image_repo="${target_release_image%:*}"
+target_release_image_repo="${target_release_image_repo%@sha256*}"
+
+haproxy_image_pullspec=$(oc adm release info "${RELEASE_IMAGE_LATEST}" --image-for haproxy-router | awk -F'@' '{print $2}')
+tgt_haproxy_image="${target_release_image_repo}@${haproxy_image_pullspec}"
+echo "target haproxy image: ${tgt_haproxy_image}"
+
+cat > "${SHARED_DIR}/haproxy-router-image" << EOF
```

Comment:
ipi disconnected install need this file? I did not see somewhere referent it.

Reply:
IPI disconnected install doesn't need this file, only used to upi disconnected install

Reply:
maybe need to write separate ref for haproxy config update? I thought ipi disconnected installation doesn't have impact even if calling this part, since as you said, no referent it in later steps.

Reply:
1. I saw ipi disconnect workflow also reference it. 
2. I have some other idea about saving bastion info. maybe we can talk on slack.